doc/conf.py

```diff
@@ -2,9 +2,8 @@
 This configuration only affects single-package Sphinx documentation builds.
 """
 
-from documenteer.conf.pipelinespkg import *  # noqa
-
 import lsst.ts.ataos  # noqa
+from documenteer.conf.pipelinespkg import *  # noqa
```

Comment:
what is the #noqa?

Reply:
I think it's to ignore the checks in this line. I think it's because `import *` violates some flake8 check.